src/api/task/api/dto/task.go

```diff
@@ -1,51 +1,62 @@
 package dto
 
 import (
+	"proyectos/src/api/constants"
+	"proyectos/src/api/errors"
 	"proyectos/src/api/task/domain/model"
 	"time"
 )
 
 // Task dto
 type Task struct {
 	ID             int64     `validate:"gt=0" json:"id"`
-	Name           string    `validate:"required,min=2,max=50" json:"name"`
+	Name           string    `validate:"required,min=2,max=500" json:"name"`
 	Description    string    `validate:"required" json:"description"`
 	StartDate      time.Time `validate:"required" json:"start_date"`
-	HoursWorked    int       `validate:"required" json:"hours_worked"`
+	WorkedHours    int       `validate:"required" json:"worked_hours"`
```

Comment:
Why is hours worked a required field? Theoretically we should initialize it at 0 hours upon creation and only modify it when work is logged

Reply:
shrek

Reply:
como en la otra consulta, son datos que tienen que estar, me parece bien que al momento de crearse se defaultee a 0